src/components/campus-toggle/campus-toggle.component.tsx

```diff
@@ -1,12 +1,6 @@
 import React, { useState } from "react";
-import {
-  StyleSheet,
-  View,
-  Text,
-  Dimensions,
-  TouchableHighlight,
-} from "react-native";
-import { CampusId, Region } from "../../types/main";
+import { StyleSheet, View, Text, TouchableHighlight } from "react-native";
+import { Region } from "../../types/main";
 
 import { getCampusById } from "../../constants";
```

Comment:
Inconsistent with how it's imported in `user-story-2.spec`

Reply:
In consistent in what way? Here is how its imported in `user-story-2.spec.ts`:

`import { getCampusById } from "../src/constants";`